saleor/graphql/payment/types.py

```diff
@@ -81,82 +86,42 @@ class Meta:
 
 class Payment(ModelObjectType):
     id = graphene.GlobalID(required=True)
+    gateway = graphene.String(required=True)
+    is_active = graphene.Boolean(required=True)
+    created = graphene.DateTime(required=True)
+    modified = graphene.DateTime(required=True)
```

Comment:
IMO, we should use `created_at` and `modified_at` as we decided to choose this instead of `created` and `modified`.

Reply:
It is not exactly a new field.  Because I've added a separate model for a new way of handling payments, I've reverted back the payment model to the structure as it was previously(main). This part will bring some conflicts after your branch with changes created->createdAt will be merged. But I would prefer to resolve them there instead of modifying the same model's fields as  you did in your PR